requirements_all.txt

```diff
@@ -171,6 +171,7 @@ freesms==0.1.1
 
 # homeassistant.components.device_tracker.fritz
 # homeassistant.components.sensor.fritzbox_callmonitor
+# homeassistant.components.sensor.fritzbox_callforwarding
```

Comment:
How come this is "sensor" ? Please don't update this file manually but follow the instructions in the PR.